lib/sqlalchemy/dialects/mysql/mariadb.py

```diff
@@ -7,13 +7,26 @@
 # mypy: ignore-errors
 from .base import MariaDBIdentifierPreparer
 from .base import MySQLDialect
+from .base import MySQLTypeCompiler
+
+
+class MariaDBTypeCompiler(MySQLTypeCompiler):
+    def visit_uuid(self, type_, **kw):
+        if (
```

Comment:
**Michael Bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5120